source/index-management/tests/README.rst

```diff
@@ -198,3 +198,26 @@ Case 5: ``dropSearchIndex`` suppresses namespace not found errors
 
 #. Create a driver-side collection object for a randomly generated collection name.  Do not create this collection on the server.
 #. Run a ``dropSearchIndex`` command and assert that no error is thrown.
+
+Case 6: Driver can successfully create and list search indexes with non-default readConcern and writeConcern
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Create a collection with the "create" command using a randomly generated name (referred to as ``coll0``).
+#. Apply a write concern ``WriteConcern(w=1)`` and a read concern with ``ReadConcern(level="local")`` to ``coll0``.
```

Comment:
During my testing, I noticed that the server rejects everything but 'local'` ReadConcern` for the `$listSearchIndexes `stage. Should we consider using any other `ReadConcern` to avoid potential false positives in the test?

Reply:
Great catch. Updated the prose test to use `majority`. If `majority` is applied as the `readConcern`, the test fails with `$listSearchIndexes cannot run with a readConcern other than 'local'. Current readConcern: majority`.